app/src/components/LoadingScreen.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ActivityIndicator, StyleSheet, View } from 'react-native';
+
+export type Props = { delayMs?: number };
+
+const LoadingScreen: React.FC<Props> = ({ delayMs = 500 }) => {
+  const [doAnimate, setDoAnimate] = React.useState<boolean>(false);
+
+  React.useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setDoAnimate(true);
+    }, delayMs);
+
+    return () => {
+      clearTimeout(timeoutId);
+    };
+  }, [delayMs]);
+
+  return (
+    <View style={styles.container}>
+      {doAnimate ? <ActivityIndicator size="large" /> : null}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+    justifyContent: 'center',
+  },
+});
+
+export default LoadingScreen;
```

Comment:
This is called `LoadingScreen` but it's not under the screen folder. Any reasons why? Also, instead of calling it `LoadingScreen`, can we remove the `Screen` part to be consistent with other naming? For example, we don't call `Home` screen `HomeScreen` :)

Reply:
Good question. It's not a screen at all, maybe just `Loading` then?

Reply:
How about `Busy` or `Spinner`? I prefer `Busy`. But if you have strong opinion on `Loading`, then we can go with that.